EnigmaAutomation/__init__.py

```diff
@@ -1,3 +1,3 @@
-from .celery import app as celery_app
+from .celery import app as celery_app # noqa
```

Comment:
lets not add inline `noqa`

Reply:
Removed all `noqa` from `enigma_automation`  folder.